README.md

```diff
@@ -60,15 +60,26 @@ docker-machine create -d ovh node-1
 |``--ovh-consumer-key`` or ``$OVH_CONSUMER_KEY``            |Consumer Key      |none      |yes|
 |``--ovh-region``                                           |Cloud region      |GRA1      |no|
 |``--ovh-flavor``                                           |Cloud Machine type|vps-ssd-1 |no|
+|``--ovh-image``                                            |Cloud Machine image|Ubuntu 16.04 |no|
+|``--ovh-ssh-user``                                         |Cloud Machine SSH User|ubuntu |no|
```

Comment:
I'd be tempted to mark this parameters as ``(advanced) Cloud Machine ...`` to avoid use by mistake.